Week-12/queries.sql

```diff
@@ -0,0 +1,17 @@
+/* Find the item that has minimum weight */
+select * from items order by itemWeight asc limit 1;
+
+/* Find the different warehouses in “Pune”.*/
+select * from warehouse inner join city where city.cId=warehouse.cId and cityName='Pune';
```

Comment:
can we use inner join without `on` ?? @Salmanz18

Reply:
My bad, I missed writing `on` in this query.